### ISSUE TITLE
Recent versions of pull-stream break WebSocket plugin

### ISSUE THREAD

**Issue by tlrobinson** [ssbc/multiserver#10]
It looks like `require('pull-stream/pull')` no longer has `map`, but `require('pull-stream')` does.

```
/node_modules/multiserver/plugins/ws.js:34
          stream.source = pull(stream.source, pull.map(Buffer))
                                                   ^

TypeError: pull.map is not a function
    at onConnect (/node_modules/multiserver/plugins/ws.js:34:52)
    at WebSocket.<anonymous> (/node_modules/pull-ws/source.js:49:13)
    at WebSocket.onError (/node_modules/ws/lib/WebSocket.js:452:14)
    at emitOne (events.js:116:13)
    at WebSocket.emit (events.js:211:7)
    at ClientRequest.onerror (/node_modules/ws/lib/WebSocket.js:711:10)
    at emitOne (events.js:116:13)
    at ClientRequest.emit (events.js:211:7)
    at Socket.socketErrorListener (_http_client.js:387:9)
    at emitOne (events.js:116:13)
```

**Comment by dominictarr** [comment]
fixed in 1.10.1

**Comment by tlrobinson** [comment]
Thanks, confirmed.
